src/conversions/std/cell.rs

```diff
@@ -22,6 +22,18 @@ impl<'py, T: Copy + IntoPyObject<'py>> IntoPyObject<'py> for Cell<T> {
     type Output = T::Output;
     type Error = T::Error;
 
+    #[inline]
+    fn into_pyobject(self, py: Python<'py>) -> Result<Self::Output, Self::Error> {
+        self.get().into_pyobject(py)
+    }
+}
+
+impl<'a, 'py, T: Copy + IntoPyObject<'py>> IntoPyObject<'py> for &'a Cell<T> {
```

Comment:
This one is interesting; because of the nature of the cell as a copy-on-read container it means that we have `T: IntoPyObject` instead of `&T`, despite this being a reference to a container.

If we wanted to keep it consistent with the `ToPyObject` implementation above (and the general trend), this implementation should probably go through `&T`. But is that actually correct, or was the original `ToPyObject` implementation just wrong? 🤔

Reply:
Interesting thought... I'd argue that this implementation makes more sense, since we get an owned `T` out. There would be nothing to tie the lifetime of that reference to (since it is a temporary inside the impl) and we would have to write HRTB to express that. In the end it basically describes the same thing, with a lot more syntax. 

I think in the `ToPyObject` is masks this HRTB kind of because the implementation can't define or restrict the "`'self`" lifetime and has to be valid for any lifetime.

Reply:
Yes, I think I agree that this new implementation is the correct formulation 👍